### PR TITLE
LPS-143340 Add unit test to cover the interpret method

### PR DESCRIPTION
**Pull request by seltong**

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-task-web/src/main/java/com/liferay/portal/workflow/task/web/internal/notifications/WorkflowTaskUserNotificationHandler.java [hunk 1]
```diff
@@ -53,6 +59,38 @@ public WorkflowTaskUserNotificationHandler() {
...
+		throws PortalException {
+
+		JSONObject jsonObject = JSONFactoryUtil.createJSONObject(
+			userNotificationEvent.getPayload());
+
+		WorkflowHandler<?> workflowHandler =
+			WorkflowHandlerRegistryUtil.getWorkflowHandler(
+				jsonObject.getString("entryClassName"));
+
+		if (workflowHandler != null) {
```
Comment (MarinhoFeliphe) [inline modules/apps/portal-workflow/portal-workflow-task-web/src/main/java/com/liferay/portal/workflow/task/web/internal/notifications/WorkflowTaskUserNotificationHandler.java:75]: Hi @seltong, please set this condition inline.

Thanks!

File: modules/apps/portal-workflow/portal-workflow-task-web/src/main/java/com/liferay/portal/workflow/task/web/internal/notifications/WorkflowTaskUserNotificationHandler.java [hunk 2]
```diff
@@ -53,6 +59,38 @@ public WorkflowTaskUserNotificationHandler() {
...
+			_BODY_TEMPLATE_DEFAULT, new String[] {"[$BODY$]", "[$TITLE$]"},
+			new String[] {
+				LanguageUtil.format(
+					locale, "notification-for-x-was-inactivated",
+					jsonObject.getString("entryType"), false),
+				LanguageUtil.get(locale, "notification-no-longer-applies")
+			});
+
+		return new UserNotificationFeedEntry(
+			false, body, StringPool.BLANK, false);
```
Comment (MarinhoFeliphe) [inline modules/apps/portal-workflow/portal-workflow-task-web/src/main/java/com/liferay/portal/workflow/task/web/internal/notifications/WorkflowTaskUserNotificationHandler.java:91]: Hi @selton, define body variable inline, please.

Thanks!

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by seltong** [comment]
ci:forward

**Pull request closed without merging**